05_data_pipeline/generate_data.py

```diff
@@ -24,131 +45,178 @@ def __init__(self):
         self.answer_generator = AnswerGenerator()
 
     def forward(self, x):
-        """Defines the pipeline
-        
-        forward() is invoked by __call__() as shown in run_pipeline() below.
-        """
-        x = self.question_generator(x, output_type={
-            "question_1": "str",
-            "question_2": "str",
-            "question_3": "str",
-        })
+        x = self.question_generator(x)
         x = self.answer_generator(x)
         return x
 
-def get_company_info(chunk):
-    info = f"Company: {chunk.data['exchange']}\n"
-    info += f"Ticker: {chunk.data['ticker']}\n"
-    info += f"Date: {chunk.data['date']}\n"
-    info += f"Quarter: {chunk.data['q']}\n"
-    return info
 
 class QuestionGenerator(GenerationNode):
-    """GenerationNode represents a step of processing in a pipeline.
```

Comment:
Probably leave these comments, if you intend to merge this PR.
These comments are meant for readers of example code to easier understand the code.